src/python/pants/engine/internals/mapper_test.py

```diff
@@ -153,7 +153,8 @@ def test_duplicate_names(self) -> None:
             )
 
 
-HydratedStructs = Collection[HydratedStruct]
+class HydratedStructs(Collection[HydratedStruct]):
+    pass
```

Comment:
You beat me to it. I have two separate PRs to fix this, but haven't been able to get either green. Glad to see it finally fixed here :)